src/features/activities/activity-form/ActivityEditForm.tsx

```diff
@@ -1,38 +1,37 @@
-import FullscreenModal from '@modals/FullscreenModal';
-import { X } from '@phosphor-icons/react';
-import { ChangeEvent, MouseEvent, TouchEvent, useEffect } from 'react';
-import Alert from '../../../components/generic/Alert';
-import Button from '../../../components/input/button/Button';
-import DatePicker from '../../../components/input/date-picker/DatePicker';
-import IconPicker from '../../../components/input/icon-picker/IconPicker';
-import RatingPicker from '../../../components/input/rating-picker/RatingPicker';
-import TextField, { TextFieldElement } from '../../../components/input/text-field/TextField';
-import TimePicker from '../../../components/input/time-picker/TimePicker';
-import { db } from '../../../data/Database';
-import CategorySelect from '../../categories/category-selection/CategorySelect';
-import { Activity } from '../Activity.interface';
-import classes from './ActivityEditForm.module.scss';
-import {
-  DELETE_GUARD_ALERT,
-  VALIDATION_ALERTS
-} from './state/activityForm.constants';
-import useActivityForm from './state/useActivityForm';
+import React, { MouseEvent, TouchEvent, useEffect } from "react";
+import FullscreenModal from "@modals/FullscreenModal";
+import { X } from "@phosphor-icons/react";
+import Alert, { useAlerts } from "../../../components/generic/Alert";
+import Button from "../../../components/input/button/Button";
+import DatePicker from "../../../components/input/date-picker/DatePicker";
+import IconPicker from "../../../components/input/icon-picker/IconPicker";
+import RatingPicker from "../../../components/input/rating-picker/RatingPicker";
+import TextField from "../../../components/input/text-field/TextField";
+import TimePicker from "../../../components/input/time-picker/TimePicker";
+import { db } from "../../../data/Database";
+import CategorySelect from "../../categories/category-selection/CategorySelect";
+import { Activity } from "../Activity.interface";
+import classes from "./ActivityEditForm.module.scss";
+import { DELETE_GUARD_ALERT, VALIDATION_ALERTS } from "./state/activityForm.constants";
+import useActivityForm from "./state/useActivityForm";
 
 interface ActivityEditFormProps {
   onClose: () => void;
   activity?: Partial<Activity>;
   onCloseTemplateSelection?: () => void;
 }
 
-const CLOSE_ICON_PROPS = {
-  size: 24,
-};
+type SeverityType = "success" | "error" | "info" | "warning";
```

Comment:
Rather than using strings, an enum would be more effective. It keeps numeric values which significantly reduced memory usage.